cfpb/sbl-frontend
# [Page] [Provide Filing Details] Rename Files and Update Tests

Moving/Renaming Files and updating Tests

## Changes

- Found appropriate instances of "point of contact" and renamed them to "filing details"
- Moved files from point of contact directories to filing details directories
- Updated test fixture to click a voluntary reporter status

## How to test this PR

1. Pull the branch in sbl-frontend
2. Make sure that the following additional projects are up to date
    - regtech-cleanup-api
    - regtech-mail-api
    - regtech-user-fi-management
    - sbl-filing-api
    - sbl-project
4. Restart the stack as necessary
5. Navigate from/to filing details and verify that things are navigating correctly
6. Run the e2e tests and verify that they pass with the exception of the those mentioned in the notes
